core/services/database/WordPressOption.php

```diff
@@ -32,36 +34,47 @@ abstract class WordPressOption
     /**
      * @var boolean
      */
-    private $autoload;
+    private $autoload = false;
 
     /**
      * @var mixed
      */
-    private $default_value;
+    private $default_value = null;
 
     /**
      * @var string
      */
-    private $option_name;
+    private $option_name = '';
 
     /**
      * @var mixed
      */
     private $value = WordPressOption::NOT_SET_YET;
 
+	/**
+	 * @var OptionEngine
+	 */
+	private $option_engine;
+
 
     /**
      * WordPressOption constructor.
      *
-     * @param bool   $autoload
-     * @param mixed  $default_value
      * @param string $option_name
+     * @param mixed  $default_value
+     * @param bool   $autoload				if true, will load the option on EVERY request
+     * @param bool 	 $is_network_option		if true, will save the option to the network as opposed to the current blog
      */
-    public function __construct(string $option_name, $default_value, bool $autoload = false)
-    {
-        $this->setAutoload($autoload);
+    public function __construct(
+		string $option_name,
+		$default_value,
+		bool $autoload = false,
+		bool $is_network_option = false
+	) {
+		$this->setAutoload($autoload);
```

Comment:
Tabs here.

Reply:
ya phpStorm started doing this recently despite tabs being turned off

<img width="279" alt="Screenshot 2022-05-05 001035" src="https://user-images.githubusercontent.com/1751030/166877108-e937029b-b876-41fe-b805-feab14a50668.png">

it's driving me crazy, which is totally not necessary when I'm already within walking distance of crazy

Reply:
I hear you !

Reply:
figured it out... it was an `.editorconfig` file in the root of my project